comp/core/config/go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	github.com/DataDog/datadog-agent/pkg/config/setup v0.52.0-rc.3
 	github.com/DataDog/datadog-agent/pkg/util/fxutil v0.52.0-rc.3
 	github.com/DataDog/datadog-agent/pkg/util/winutil v0.52.0-rc.3
-	github.com/DataDog/viper v1.12.0
+	github.com/DataDog/viper v1.12.1-0.20240311110542-3e7837fd388f
```

Comment:
Should we tag DataDog/viper with 1.13.0 ?

Reply:
Sure !

Reply:
Done !